ui/app/routes/variables/variable.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+import withForbiddenState from 'nomad-ui/mixins/with-forbidden-state';
+import WithModelErrorHandling from 'nomad-ui/mixins/with-model-error-handling';
+
+export default class VariablesVariableRoute extends Route.extend(
+  withForbiddenState,
```

Comment:
`nit`: I'm noticing this is. `camelCase` instead of  `PascalCase`.

Reply:
Good catch; resolved.